charts/ups-broker/templates/broker.yaml

```diff
@@ -0,0 +1,8 @@
+{{- if .Values.deployBrokerResource }}
+apiVersion: servicecatalog.k8s.io/v1alpha1
+kind: Broker
+metadata:
+  name: ups-broker
+spec:
+  url: http://{{ template "fullname" . }}.{{ .Release.Namespace }}.svc.cluster.local
+{{ end }}
```

Comment:
I am all for this, but it won't work unless the service-catalog API server is run behind the aggregator. Can you document that please?